src/components/AnalyticsTable.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { Col, Row, Table } from 'reactstrap';
+import PropTypes from 'prop-types';
+
+import MagicMarkdown from 'components/MagicMarkdown';
+
+// Data should be:
+// {
+//   type: 'table',
+//   description: str,
+//   tables: [
+//     {
+//       headers: [
+//         {
+//           header: str, label for the column
+//           key: str, key for values from this column in data
+//           rowHeader: bool, whether this column is a header for the rows
+//         }
+//       ],
+//       rows: [
+//         {
+//           [key]: str|float, value to show for column with key [key]
+//         }
+//       ],
+//     },
+//   ],
+// }
+const HeaderCell = ({ children, ...props }) => (
+  <th scope="row" {...props}>
+    {children}
+  </th>
+);
+const RegularCell = ({ children, ...props }) => <td {...props}>{children}</td>;
+RegularCell.propTypes = { children: PropTypes.shape({}).isRequired };
```

Comment:
```suggestion
RegularCell.propTypes = { children: PropTypes.node.isRequired };
```
Children should always be proptypes.node afaik